config/environments/development.rb

```diff
@@ -42,7 +42,4 @@
 
   # Annotate rendered view with file names.
   # config.action_view.annotate_rendered_view_with_filenames = true
-
```

Comment:
This is a Rails 7.1+ specific configuration that somehow snuck in here and is breaking things in local development.  Backing it out now since we're still on Rails 7.0.8.  See: https://guides.rubyonrails.org/configuring.html#config-action-controller-raise-on-missing-callback-actions